styles.css

```diff
@@ -761,39 +834,106 @@ textarea {
     margin-top: 10px;
   }
 
-  .bio .text-area p {
+  .about-me .container2 p {
+    font-size: 22px;
+    line-height: 34px;
+    color: #f4f5f7;
+    margin-top: 24px;
+    margin-bottom: 20px;
+    width: 500px;
+    margin-left: 142px;
+    position: inherit;
+  }
+
+  .bio .text-area .intro p {
     font-family: "Poppins", sans-serif;
-    font-size: 21px;
+    font-size: 20px;
     font-weight: 300;
     line-height: 24px;
     color: #fff;
     margin-top: 15px;
+    padding-right: 480px;
   }
 
   .contact {
     display: flex;
     justify-content: space-between;
-    padding-left: 20px;
-    padding-right: 100px;
-    margin-top: 20px;
+    padding-left: 0;
+    padding-right: 0;
+    margin-top: 0;
+    margin-right: 50px;
   }
 
-  .contact .social li {
-    flex: 1;
-    justify-content: space-around;
-    padding-left: 5px;
-    padding-right: 30px;
-    margin-top: 20px;
+  .contact .social {
+    flex-direction: column;
+    padding-right: 20px;
+    margin-top: 110px;
+  }
+
+  .contact .social a {
+    flex-direction: column;
+    padding-right: 20px;
+    margin-bottom: 10px;
+  }
+
+  .about-me {
+    background: #1c1a19;
+    width: 100%;
   }
 
   .social li a i {
     align-items: center;
   }
 
+  .container .hr {
+    height: 83px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    width: 700px;
+    display: flex;
+    color: #1c1a19;
+    border-bottom: 0;
+    margin-bottom: 0;
+    padding-bottom: 24px;
+    margin-top: 120px;
+    padding-right: 60px;
+    position: relative;
+  }
+
+  .container .hr::after {
+    content: "";
+    display: block;
+    background: #212121;
+    width: 430px;
+    height: 1px;
+    position: absolute;
+    top: 40%;
+    right: 0;
+  }
+
   .works .container {
     padding: 0 10px;
   }
 
+  .works .container .project {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
```

Comment:
- [ ] Please use `grid` instead of `flex` because `CSS Grid Layout` excels at dividing a page into major regions or defining the relationship in terms of `size`, `position`, and `layer`, between parts of a control built from `HTML` primitives. Like `tables`, `grid layout` enables an author to align elements into `columns` and `rows`. Also, This is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) . :+1: 
- [ ] Also, your design is not the same as the Figma design as I mentioned above :point_up: please stick to the Figma design. Check :point_down: below for the comparison between your design and Figma design and kindly fix these issues. :slightly_smiling_face:  

**Your Design**
![screencapture-127-0-0-1-5501-2023-03-01-19_35_41 (1)](https://user-images.githubusercontent.com/45226848/222179639-51bab428-ca25-4e54-b2f3-b55d8d1e2e25.png)

**Figma Design**
![Screenshot from 2023-03-01 19-43-24](https://user-images.githubusercontent.com/45226848/222179719-07685a6b-1ad8-43d3-9630-5bd075c43be6.png)

- [ ] If you notice the space between `My Recent Works` and `Cards` these space is not the same as the Figma design kindly reduce it. :+1:  
- [ ] If Check your `Cards` in your case it's four `Cards` are in one `row` and two `Cards` are in the second `row` but in Figma design three `Cards` are in one `row` and other three `Cards` are in next row kindly fix these issues. :+1: